.github/workflows/shadow-push.yml

```diff
@@ -0,0 +1,29 @@
+name: shadow-push
+
+on: push
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    env:
+      API_TOKEN_GITHUB: ${{ secrets.API_TOKEN_GITHUB }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set env vars
+        run: |
+          echo "BRANCH_NAME=${GITHUB_REF##*/}" >> $GITHUB_ENV
+          echo NODE_VERSION=`cat .nvmrc` >> $GITHUB_ENV
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: 'yarn'
+      - run: yarn install
+      - run: yarn build
```

Comment:
@jerivas Why'd you decide to go with this instead of `yarn prod`?

Reply:
```
"prod": "ELEVENTY_ENV=production run-s update-oss-docs build",
```
It seemed to me like `prod` just runs extra stuff on top of `build`, so I assumed it would be slower. We can switch to `prod` if we want to capture `update-oss-docs`, or even add a separate script tailored for diffing

Reply:
Yeah, I suppose you're right. We could run `ELEVENTY_ENV=production build`, but this is probably good enough as-is.